### DISCUSSION TITLE
[Question]: What is the correct usage for the following arrow icons?

### DISCUSSION THREAD

**Discussion by kunderwoodatadaptive** [carbon-design-system/carbon#13299] [Help] [answered]
### Question for Carbon

Hi,

I'm trying to familiarize myself with the awesome icons that are included in Carbon.

Do you offer usage definitions for the following icons?

- Arrows--horizontal
- Arrows--vertical
- Caret--sort
- Chevron--sort
- Choose-item
- Drag--vertical
- Arrow--right
- Arrow—down
- Arrow—left
- Arrow--up-left
- Arrow--down-right

Thank you for your help!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)

**Comment by andreancardona** [comment]
@laurenmrice also, curious on your thoughts, on this? Thank you!!

**Comment by laurenmrice** [comment, marked as answer]
Hi @kunderwoodatadaptive 👋 

We mostly leave it up to the designer to determine the usage of the icon and where it will be placed in a UI. However there are some common use cases where these icons show up:

`Arrows--vertical`
- Carbon uses this as a sorting icon to sort data table column headers along with the `arrow--up` and `arrow--down` icons.
<img width="1185" alt="Screen Shot 2023-02-28 at 12 37 29 PM" src="https://user-images.githubusercontent.com/43969356/221933133-26a76e48-dfd0-403a-a095-b51425cbc573.png">

`Caret--sort`
- Carbon used this icon as our default for our number input stepper. We now use the `subtract` and `add` icons as steppers instead.

`Chevron--sort`
- This can be used for a similar use case as listed above^.

`Drag--vertical`
- Use cases where this has been used is in data tables with rows that you can reposition and drag to another spot above or below inside of the table. This also has been used to drag cards up and down to reposition them on a page.

`Arrow--right`
- RIght arrows are usually used in buttons, links, and in tiles/cards.

<img width="878" alt="Screen Shot 2023-02-28 at 12 45 59 PM" src="https://user-images.githubusercontent.com/43969356/221935108-3d3f562c-448a-46e6-a51f-701d0cf31e4b.png">
<img width="1017" alt="Screen Shot 2023-02-28 at 12 46 40 PM" src="https://user-images.githubusercontent.com/43969356/221935252-af1955d0-ca93-45f2-91eb-d15c03690baa.png">

`Arrow--left`
- Has been used as a back arrow paired with a link at the top of a page to go back to the previous page.

`Arrow--up-left` and `Arrow--down-right`
- These kinds of arrows can be used to represent data that is increasing/decreasing.

<img width="285" alt="Screen Shot 2023-02-28 at 12 47 06 PM" src="https://user-images.githubusercontent.com/43969356/221935353-668d85e7-67cc-4252-b34c-7c245e0b9151.png">

**Comment by kunderwoodatadaptive** [comment]
@laurenmrice , that's very helpful, thank you!

**Comment by kunderwoodatadaptive** [comment]
Hi, I have a few remaining icons I'd like information about:

1. Next--filled
2. Caret--right
3. Caret--down
4. Choose--item
5. Chevron--right
6. Chevron--down
7. Settings--adjust

Thank you for your help!

**Comment by laurenmrice** [comment]
@kunderwoodatadaptive 
- For the first 6, I would use these icons in cases where they intuitively make sense to you. On the Carbon side, we don't have any strict guidance or use cases of when to use them.

- For the `Settings--adjust` icon is typically used to adjust settings for something, like a data table.
